internal/pkg/log/fields.go

```diff
@@ -18,7 +18,9 @@ const (
 	FieldUserLogLevel        = "userLogLevel"
 	FieldID                  = "id"
 	FieldName                = "name"
+	FieldOut                 = "out"
```

Comment:
I think this should be more descriptive. What is actually being logged here?

Reply:
It's mainly for docker-compose command  in bddtests-test.go, in this PR it's at line 130 right now.

Reply:
So, maybe WithDockerResponse or if you want something more generic, WithCommandResponse?

Reply:
i'll rename to command instead

Reply:
done